examples/dreambooth/train_dreambooth_lora_hidream.py

```diff
@@ -1140,7 +1131,7 @@ def main(args):
     if args.lora_layers is not None:
         target_modules = [layer.strip() for layer in args.lora_layers.split(",")]
     else:
-        target_modules = ["to_k", "to_q", "to_v", "to_out.0"]
+        target_modules = ["to_k", "to_q", "to_v", "to_out"]
```

Comment:
Perhaps we can add a comment explaining that including `to_out` will target all the expert layers.